modules/notifications/notifications.php

```diff
@@ -1223,25 +1259,53 @@ function settings_post_types_option() {
 	 */
 	function settings_always_notify_admin_option() {
 		$options = array(
-			'off' => __( 'Disabled', 'edit-flow' ),			
+			'off' => __( 'Disabled', 'edit-flow' ),
 			'on' => __( 'Enabled', 'edit-flow' ),
 		);
 		echo '<select id="always_notify_admin" name="' . $this->module->options_group_name . '[always_notify_admin]">';
 		foreach ( $options as $value => $label ) {
 			echo '<option value="' . esc_attr( $value ) . '"';
-			echo selected( $this->module->options->always_notify_admin, $value );			
+			echo selected( $this->module->options->always_notify_admin, $value );
+			echo '>' . esc_html( $label ) . '</option>';
+		}
+		echo '</select>';
+	}
+
+	/**
+	 * Option to enable sending notifications to Slack
+	 *
+	 * @since 0.9.9
+	 */
+	function settings_send_to_slack() {
+		$options = array(
+			'off' => __( 'Disabled', 'edit-flow' ),
+			'on' => __( 'Enabled', 'edit-flow' ),
+		);
+		echo '<select id="send_to_slack" name="' . $this->module->options_group_name . '[send_to_slack]">';
+		foreach ( $options as $value => $label ) {
+			echo '<option value="' . esc_attr( $value ) . '"';
+			echo selected( $this->module->options->send_to_slack, $value );
 			echo '>' . esc_html( $label ) . '</option>';
 		}
 		echo '</select>';
 	}
 
+	/**
+	 * Option to set the Slack webhook URL
+	 *
+	 * @since 0.9.9
+	 */
+	function settings_slack_webhook_url() {
+		echo '<input type="text" id="slack_webhook_url" name="' . $this->module->options_group_name . '[slack_webhook_url]" value="' . esc_attr( $this->module->options->slack_webhook_url ) . '" />';
```

Comment:
```suggestion
		echo '<input type="text" id="slack_webhook_url" name="' . esc_attr( $this->module->options_group_name ) . '[slack_webhook_url]" value="' . esc_attr( $this->module->options->slack_webhook_url ) . '" />';
```

Reply:
Changed in 8bf4f9f190d771d89f4cdaa2855b5b37929f9f63